pod/perlfunc.pod

```diff
@@ -3896,11 +3896,13 @@ X<lc> X<lowercase>
 
 =for Pod::Functions return lower-case version of a string
 
-Returns a lowercased version of EXPR.  This is the internal function
-implementing the C<\L> escape in double-quoted strings.
+Returns a lowercased version of EXPR.
 
 If EXPR is omitted, uses L<C<$_>|perlvar/$_>.
```

Comment:
This and the previous paragraph can probably be combined now since they are about the same thing.